### PR TITLE
Deposit Feature Added

### PR DESCRIPTION
**Pull request by MuthuKumari-P**

### CODE REVIEW

File: src/main/java/in/muthukumari/controller/ExistCustomerController.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	}
+
+	/**
+	 * This method used to get the customer name and account number list
+	 * 
+	 * @return
+	 * @throws ServiceException 
+	 */
+	public List<CustomerBankDetail> getNameAndAccNumList() {
+		List<CustomerBankDetail> nameAndAccNumList = null;
```
Comment (Christykumar) [inline src/main/java/in/muthukumari/controller/ExistCustomerController.java:25]: you can use short names here like list

File: src/main/java/in/muthukumari/servlet/CheckBalanceServlet.java [hunk 2]
```diff
@@ -34,15 +33,12 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
 			List<Long> accNum = BankService.getAccountNumber(userName);
 			if (!accNum.isEmpty()) {
 				session.setAttribute("accNum", accNum);
-				RequestDispatcher rd = request.getRequestDispatcher("checkbalance.jsp");
-				rd.forward(request, response);
+				response.sendRedirect("checkbalance.jsp");
 			} else {
-				String infoMsg = "You doesn't create your account!!! So, Please create your account using bank details";
+				String infoMsg = "Sorry! You doesn't have an account!!! So, Please create your account using bank details";
```
Comment (Christykumar) [inline src/main/java/in/muthukumari/servlet/CheckBalanceServlet.java]: dosen't- don't

File: src/main/java/in/muthukumari/controller/ExistCustomerController.java (hunk 1 above)
Comment (MuthuKumari-P) [reply src/main/java/in/muthukumari/controller/ExistCustomerController.java:25]: I have changed this

File: src/main/java/in/muthukumari/servlet/CheckBalanceServlet.java (hunk 2 above)
Comment (MuthuKumari-P) [reply src/main/java/in/muthukumari/servlet/CheckBalanceServlet.java]: I have changed this.

**Pull request merged**